### ISSUE TITLE
Broken links to IPEA gpkg files

### ISSUE THREAD

**Issue by pinho** [loggi/loggibud#24]
All links to download geographic data from IPEA is broken returning 404 (Not Found).

Hence, the generation pipeline described in [instance generation README.md](https://github.com/loggi/loggibud/blob/master/loggibud/v1/instance_generation/README.md) finish unsuccessfully. The `generate.py` script fails to find the `*.gpkg` files in the `data_raw` directory, raising following error:

```
fiona._err.CPLE_OpenFailedError: ./loggibud/loggibud/v1/instance_generation/../../../data_raw/33.gpkg: No such 
file or directory
```

The (broken) links used in `download.sh` script are:

* https://www.ipea.gov.br/geobr/data_gpkg/census_tract/2010/33.gpkg
* https://www.ipea.gov.br/geobr/data_gpkg/census_tract/2010/53.gpkg
* https://www.ipea.gov.br/geobr/data_gpkg/census_tract/2010/15.gpkg

**Comment by gabisurita** [comment]
Thanks for the report. I believe IPEA changed the path internally. I'll investigate this further.

**Comment by minterciso** [comment]
I'm also experiencing this issue, will this be updated soon?

**Comment by gabisurita** [comment]
By checking the metadata from IPEA ([available at this CSV](http://www.ipea.gov.br/geobr/metadata/metadata_gpkg.csv)), I believe that the new paths are:

https://www.ipea.gov.br/geobr/data_gpkg/census_tract/2010/33census_tract_2010.gpkg
https://www.ipea.gov.br/geobr/data_gpkg/census_tract/2010/53census_tract_2010.gpkg
https://www.ipea.gov.br/geobr/data_gpkg/census_tract/2010/15census_tract_2010.gpkg

But I couldn't validate if anything else changed, and the rest of the pipeline still works.

**Comment by pinho** [comment]
@gabisurita I used these links to manually download the files (using curl) and placed them into the `data_raw` directory. The instances were generated successfully.

Therefore, I believe that replace the links in "download.sh" fix this issue. I can test this and open a PR if all OK.

**Comment by gabisurita** [comment]
Solved by #28 .

**Comment by gabisurita** [comment]
We should also fix the README links, but I'll keep this closed since the script is already working.
